packages/image-to-png/src/index.ts

```diff
@@ -141,8 +153,15 @@ export class ImageToPng {
       const base64 = image.split(',')[1];
       let buffer = Buffer.from(base64, 'base64');
       const png = PNG.sync.read(buffer);
-      if (png.interlace) {
-        buffer = PNG.sync.write(png, { interlace: false });
+      if (png.interlace || png.colorType != 2) {
```

Comment:
This is the only place that fix should be in. All the PNGs will go through this method.